### DISCUSSION TITLE
Backup successful,but send notification is failed.

### DISCUSSION THREAD

**Discussion by lengjunbin-trec** [spatie/laravel-backup#1583] [Q&A]
![image](https://user-images.githubusercontent.com/61955063/174506757-f24c1c0a-cdae-482d-aac7-cc85d6cf086e.png)
it throw error.`Serialization of 'Closure' is not allowed`,I check the logic.maybe the resuce function has problem.
![image](https://user-images.githubusercontent.com/61955063/174506864-80571843-3f6a-4b01-ad71-dc2e4d93ce4a.png)

environment: PHP 7.4.24  laravel 7       "spatie/laravel-backup": "^6.16"

**Comment by erikn69** [comment]
laravel-backup 6.x has no support 
Only 8.x has support
